libs/api/src/main/java/de/cyface/api/MeasurementRetrievalStrategy.java

```diff
@@ -68,8 +68,8 @@ public interface MeasurementRetrievalStrategy {
     default MetaData metaData(final JsonObject document) {
         final var metaData = document.getJsonObject("metaData");
         final var version = metaData.getString("version");
-        Validate.isTrue(version.equals(MetaData.CURRENT_VERSION),
-                "Encountered data in invalid format. Only Cyface Format Version 1.0.0 is supported!");
+        Validate.isTrue(version.matches(MetaData.SUPPORTED_VERSIONS),
+                "Encountered data in an unsupported deserialized format: %s", version);
```

Comment:
All deserialized data with the same API version should be supported. Right now we only have 2.0.0 in V2.